en/flight_controller/omnibus_f4_sd.md

```diff
@@ -135,27 +135,41 @@ Here is an example implementation. I used a Spektrum plug to get 3.3v from the D
 ![Omnibus F4 SD Pullup Implementation](../../assets/flight_controller/omnibus_f4_sd/pullup.jpg)
 
 
-## CRSF Crossfire Telemetry
-In addition to the [FrSky Telemetry](../peripherals/frsky_telemetry.md), the Omnibus also supports TBS CRSF Crossfire telemetry.
-It is used to send telemetry data from the flight controller (the vehicles's attitude, battery, flight mode and GPS data) to the RC transmitter (Taranis).
+## RC Telemetry
 
-Benefits are:
+The Omnibus supports telemetry to the RC Transmitter using [FrSky Telemetry](../peripherals/frsky_telemetry.md) or [CRSF Crossfire Telemetry](#crsf_telemetry).
+
+
+### CRSF Crossfire Telemetry {#crsf_telemetry}
+
+TBS CRSF Crossfire telemetry is used to send telemetry data from the flight controller (the vehicle's attitude, battery, flight mode and GPS data) to the RC transmitter (Taranis).
+
+Benefits over FrSky telemetry include:
```

Comment:
Note that these are benefits over FrSky telemetry , not benefits "in abstract" (except for the update rate) so I added that.